src/app/components/UploadImageInput/tests/UploadImageInput.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { vi } from 'vitest';
+
+import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+
+import UploadImageInput from '..';
+
+describe(`UploadImageInput`, () => {
+  afterAll(() => {
+    vi.clearAllMocks();
+  });
+
+  afterEach(() => {
+    cleanup();
+    vi.resetAllMocks();
+  });
+
+  const mockHandleFileChange = vi.fn();
+
```

Comment:
```suggestion
const id = `upload-input`;

```
this const is repeated three times so a i think this make sense

Reply:
yes, make sense!